diff --git a/js/plugins/uswds_alerts/dialogs/alertsDialog.js b/js/plugins/uswds_alerts/dialogs/alertsDialog.js
index bacc9af..524c75b 100644
--- a/js/plugins/uswds_alerts/dialogs/alertsDialog.js
+++ b/js/plugins/uswds_alerts/dialogs/alertsDialog.js
@@ -25,20 +25,20 @@ CKEDITOR.dialog.add('alertsDialog', function (editor) {
               if (this.getValue()) {
                 if (element.hasClass('usa-alert__body')) {
                   element.getParent().addClass('usa-alert--slim');
-                  element.findOne('h4').remove();
+                  element.findOne('span').remove();
                 }
                 else {
                   element.addClass('usa-alert--slim');
-                  element.findOne('h4').remove();
+                  element.findOne('span').remove();
                 }
               }
               else {
                 if (element.hasClass('usa-alert__body')) {
                   element.getParent().removeClass('usa-alert--slim');
 
-                  let header = editor.document.createElement('h4');
+                  let header = editor.document.createElement('span');
                   header.addClass('usa-alert__heading');
-                  header.setText("ALERT HEADER");
+                  header.setText("Alert Header");
 
                   element.append(header, true);
                 }
@@ -165,13 +165,13 @@ CKEDITOR.dialog.add('alertsDialog', function (editor) {
         let bodyDiv = editor.document.createElement('div');
         bodyDiv.addClass('usa-alert__body');
 
-        let header = editor.document.createElement('h4');
+        let header = editor.document.createElement('span');
         header.addClass('usa-alert__heading');
-        header.setText("ALERT HEADER");
+        header.setText("Alert Header");
 
         let body = editor.document.createElement('p');
         body.addClass('usa-alert__text');
-        body.setText('ALERT BODY TEXT');
+        body.setText('Alert body text');
 
         bodyDiv.append(header);
         bodyDiv.append(body);
